File: C#/OOP/OOP/Mercedes.cs
using System;

namespace OOP {
    class Mercedes : Car {
        public Mercedes() : base() { }

        public Mercedes(string ownerName, FuelType fuelType, Colour colour)
            : base(ownerName, fuelType, colour) { }

        public bool IsCruiseControlOn { get; set; }

        public override void GetSpecialFeature()
        {
            Console.WriteLine("I have cruise control.");
        }

        public override void Drive() {
            Console.WriteLine("Driving Mercedes");
        }
    }
}

File: Java/src/Mercedes.java
public class Mercedes extends Car {

    public Mercedes() {
        super();
    }

    public Mercedes(String ownerName, FuelType fuelType, Colour colour) {
        super(ownerName, fuelType, colour);
    }

    private boolean drivePilotOn;

    public boolean isDrivePilotOn() {
        return drivePilotOn;
    }

    public void setDrivePilotOn(boolean drivePilotOn) {
        this.drivePilotOn = drivePilotOn;
    }

    @Override
    public void getSpecialFeature() {
        System.out.println("I have drive pilot.");
    }

    @Override
    public void drive() {
        System.out.println("Driving Mercedes");
    }
}
